client/src/main/scala/drt/client/components/FlightsWithSplitsTable.scala

```diff
@@ -192,11 +193,11 @@ object FlightTableRow {
           (None, props.originMapper(flight.Origin)),
           (None, s"${flight.Gate}/${flight.Stand}"),
           (None, flight.Status),
-          (None, localDateTimeWithPopup(flight.SchDT)),
-          (None, localDateTimeWithPopup(flight.EstDT)),
-          (None, localDateTimeWithPopup(flight.ActDT)),
-          (Option("est-chox"), localDateTimeWithPopup(flight.EstChoxDT)),
-          (None, localDateTimeWithPopup(flight.ActChoxDT)),
+          (None, localDateTimeWithPopup(Some(flight.Scheduled))),
```

Comment:
It's generally bad practice to instantiate an option with Some(value), as it can result in you having Some(null). It's better, as a rule, to always use Option(value).